pmd-rules.xml

```diff
@@ -49,4 +49,4 @@
             <property name="skipAnnotations" value="true" />
```

Comment:
Remove this file so it's not part of this PR.

Reply:
Done